common/TinyLang/ParseUtils.hs

```diff
@@ -33,16 +33,18 @@ parseString parser fileName str =
     liftEither . first errorBundlePretty $ runParser parser fileName str
 
 -- Consume whitespace
-ws :: (MonadParsec e s m, Token s ~ Char) => m ()
-ws = L.space space1 empty empty
--- Last two arguments are for comment delimiters.  Let's not have any comments for now.
+ws :: (MonadParsec e s m, Token s ~ Char, Tokens s ~ [Char]) => m ()
+ws = L.space space1 lineComment blockComment where
+    -- NOTE:  Mimicking ML-family syntax comments for now
+    lineComment  = L.skipLineComment "//"
+    blockComment = L.skipBlockComment "(*" "*)"
```

Comment:
I'd prefer C-style (and Rust-style) multiline comments then: `/* <...> */`.

I'm not very familiar with ML and neither most of our users will be.

Reply:
Ok, I switched ti C-family style comments.